packages/eas-cli/src/build/configure.ts

```diff
@@ -60,6 +63,11 @@ export async function configureAsync(options: {
         "Aborting, run the command again once you're ready. Make sure to commit any changes you've made."
       );
     }
+    log.newLine();
```

Comment:
```suggestion
    log.addNewLineIfNone();
```
??

Reply:
explicit newline here was intended, I think that it looks better this way
![2020-12-01-171100_1029x293_scrot](https://user-images.githubusercontent.com/9753141/100765862-4657d780-33f8-11eb-9b57-8a7917db65cb.png)